src/libtls/tls_server.c

```diff
@@ -42,6 +43,11 @@ typedef enum {
 	STATE_FINISHED_RECEIVED,
 	STATE_CIPHERSPEC_CHANGED_OUT,
 	STATE_FINISHED_SENT,
+	/* new states in TLS 1.3 */
+	STATE_HELLORETRYREQ_SENT,
+	STATE_ENCRYPTED_EXTENSIONS_SENT,
+	STATE_CERT_VERIFY_SENT,
+	STATE_KEY_UPDATE_RECEIVED,
```

Comment:
This state is not yet used, so maybe wait with declaring it until you do.

Reply:
I agree. STATE_HELLORETRYREQ_SENT is also not yet implemented at this stage, I remove both for now.